frontend/src/helpers/vCalendarDragAndDrop.js

```diff
@@ -2,11 +2,22 @@
  * helpers for VCalendar DragAndDrop composables
  */
 
+const ONE_MINUTE = 60 * 1000
+const ONE_HOUR = 60 * ONE_MINUTE
+const ONE_DAY = 24 * ONE_HOUR
+
+const roundToMinutes = 15 // minutes
```

Comment:
It seems a little odd to have this in a constant here. Wouldn't that be better as a parameter of roundTime with a default value? Or otherwise maybe rename roundTime to roundTimeToQuarterHour?

Reply:
Renamed in #3468